deployment/ansible/test_webapp.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import requests
+
+def test_port_80_is_open(host):
+    socket = host.socket("tcp://0.0.0.0:80")
+    assert socket.is_listening
+
+def test_nginx_is_installed_and_running(host):
+    nginx = host.package("nginx")
+    assert nginx.is_installed
+    service = host.service("nginx")
+    assert service.is_running
+    assert service.is_enabled
+
+@pytest.mark.parametrize(
+    "mount_point",
+    [
+        "/",
+    ],
+)
+
+def test_path_is_mounted_core(host, mount_point):
+    mount = host.mount_point(mount_point)
+    assert mount.exists
+
+def test_app_is_responding(host):
+    response = requests.get("http://localhost:8080")
```

Comment:
```suggestion
    response = requests.get("http://localhost:80")   # nginx port, not docker
```

Reply:
**also change here** https://github.com/banigoden/Python_Web_GitHub_Actions/blob/72cedb9da3f575a3af7367d3c5fb418eb9037236/deployment/ansible/roles/deploy_web_app/templates/docker-compose.yml.j2#L7 **to expose port only for localhost**